packages/amplify-e2e-tests/src/__tests__/rds-v2.test.ts

```diff
@@ -0,0 +1,63 @@
+import { createRDSInstance, addRDSPortInboundRule, RDSTestDataProvider, removeRDSPortInboundRule, deleteDBInstance } from 'amplify-category-api-e2e-core';
+import axios from 'axios';
+import generator from 'generate-password';
+
+describe("RDS Tests", () => {
+  let publicIpCidr = "0.0.0.0/0";
+  const [db_user, db_password, db_identifier] = generator.generateMultiple(3);
+
+  beforeAll(async () => {
+    // Get the public IP of the machine running the test
+    const url = "http://api.ipify.org/";
+    const response = await axios(url);
+    publicIpCidr = `${response.data.trim()}/32`;
+  });
+
+  it("create database, setup initial tables and delete database", async () => {
+    // This test performs the below
+    // 1. Create a RDS Instance
+    // 2. Add the external IP address of the current machine to security group inbound rule to allow public access
```

Comment:
If for some reason these instances are not cleaned up, allowing public access might raise security alarms?

Reply:
We delete the security group inbound rule at the end of the test. Even for some reason if it fails to delete the rule, we allow access to one specific IP so the risk is very low. Also, we run the test in a separate account and there is no sensitive data on this account.